config_builder.go

```diff
@@ -1528,7 +1528,11 @@ func initNeutrinoBackend(ctx context.Context, cfg *Config, chainDir string,
 		PersistToDisk:      cfg.NeutrinoMode.PersistFilters,
 	}
 
-	neutrino.MaxPeers = 8
+	if cfg.NeutrinoMode.MaxPeers < defaultNeutrinoMaxPeers {
```

Comment:
what's the reason for this check? Security-related?

Reply:
it's basically a precaution to inform a user that they used an ineffective config value. I'm not sure if this is needed

Reply:
cool gotya maybe we just need to check it's greater than 0?

Reply:
right, makes sense if people want to limit below the default :+1:, updated